tests/testsearch.py

```diff
@@ -491,17 +491,48 @@ def testSequenceQuery(self):
         q1 = SequenceQuery("VLSPADKTNVKAAWGKVGAHAGEYGAEALERMFLSFPTTKTYFPHFDLSHGSAQVKGHGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPAEFTPAVHASLDKFLASVSTVLTSKYR")
         result = list(q1())
         result_len = len(result)
-        ok = result_len > 0 and result_len == 706  # this query displays 706 ids in pdb website search function
+        ok = result_len > 0  # this query displays 706 ids in pdb website search function
         logger.info("Sequence query results correctly displays ids: (%r)", ok)
 
         # Sequence query (id: 4HHB) with custom parameters
         q1 = SequenceQuery("VLSPADKTNVKAAWGKVGAHAGEYGAEALERMFLSFPTTKTYFPHFDLSHGSAQVKGHGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPAEFTPAVHASLDKFLASVSTVLTSKYR",
                            evalue_cutoff=0.01, identity_cutoff=0.9)
         result = list(q1())
         result_len = len(result)
-        ok = result_len > 0 and result_len == 313  # this query displays 313 ids in pdb website search function
+        ok = result_len > 0  # this query displays 313 ids in pdb website search function
         logger.info("Sequence query results correctly displays ids with custom parameters: (%r)", ok)
 
+    def testSeqMotifQuery(self):
+        """Test firing off a SeqMotif query"""
+        q1 = SeqMotifQuery("AA")  # basic test, make sure standard query is instantiated properly
+        result = list(q1())
+        ok = len(result) > 0
+        self.assertTrue(ok)
+        logger.info("Basic SeqMotif query results: result_length : (%d), ok : (%r)", len(result), ok)
+
+        q2 = SeqMotifQuery("FFFFF", sequence_type="dna")  # test a DNA query, this should yield no results
```

Comment:
In addition to a false test for DNA, be sure to include a positive test too (e.g., using `CCGGCG`) to ensure that the `sequence_type` of `dna` is still working.